go-controller/pkg/ovn/ovn.go

```diff
@@ -100,6 +97,10 @@ type Controller struct {
 	namespaces      map[string]*namespaceInfo
 	namespacesMutex sync.Mutex
 
+	// A cache of address sets and an associated mutex
+	addressSets     map[string]*addressSet
+	addressSetMutex *sync.RWMutex
```

Comment:
I was intending after #1244 to have all per-namespace things be in `namespaceInfo`, so there was only one set of locks to worry about...

Reply:
Per discussion today we'll keep this for now and Dave will refactor later for less locking.